tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -358,3 +469,43 @@ def _more_tags(self):
         else:
             tags['valid_oof'] = True
         return tags
+
+    def _valid_compilers(self):
+        return [RFNativeCompiler, RFOnnxCompiler]
+
+    def _get_compiler(self):
```

Comment:
Move to AbstractModel. The native compiler should be based on a new `_default_compiler(self):` method, default returns None in AbstractModel

Reply:
Following code has been added to the AbstractModel:
```python
def _default_compiler(self):
  return None
```
In RFModel, this returns RFNativeCompiler class.